acceptance/mockAgent/sampleConfig.json

```diff
@@ -0,0 +1,17 @@
+{ 
```

Comment:
If this file is no longer used (i.e. we always use the files from the data directory), then let's delete it. Otherwise, it will get confusing in the future when the files in the data directory have different kinds of properties and this file gets 'stale'